queries/scala/highlights.scm

```diff
@@ -0,0 +1,52 @@
+;; keywords
+
+[
+  "abstract"
+  "case"
+  "catch"
+  "class"
+  "def"
+  "do"
+  "extends"
+  "final"
+  "finally"
+  "forSome"
+  "implicit"
+  "lazy"
+  "macro"
+  "match"
+  "new"
+  "null"
+  "object"
+  "override"
+  "package"
+  "private"
+  "protected"
+  "sealed"
+  "super"
+  "this"
+  "throw"
+  "trait"
+  "try"
+  "type"
+  "val"
+  "var"
+  "with"
+  "yield"
+] @keyword
+
+;; special keywords
+
+[
+  "else"
+  "if"
+] @conditional
+
+[
+  "for" 
+  "while"
+] @repeat
+
+"import" @includes
```

Comment:
Can you check whether this was not `@include`?

Reply:
right again, it was `@include` not `@includes`. fixed.